app/models/proposal.rb

```diff
@@ -31,8 +31,12 @@ class Proposal < ActiveRecord::Base
   acts_as_taggable
 
   has_many :steps
-  has_many :individual_steps, ->{ individual }, class_name: 'Steps::Individual'
-  has_many :approvers, through: :individual_steps, source: :user
+  has_many :individual_steps, ->{ individual }, class_name: "Steps::Individual"
+  has_many :approval_steps, class_name: "Steps::Approval"
+  has_many :purchase_steps, class_name: "Steps::Purchase"
+  has_many :approvers_and_purchasers, through: :individual_steps, source: :user
```

Comment:
If we're using the term `step_users` elsewhere, perhaps we should use it here too? I also worry about the naming of `approvers_and_purchasers` because if/when we add `Steps::Reconcile` then this becomes `approver_and_purchasers_and_reconcilers` (oh my), etc. etc.